inc/Editor/DataBinding/BlockBindings.php

```diff
@@ -234,7 +234,7 @@ private static function get_block_fallback_content( array $source_args, array $b
 			$fallback_content = $block_attributes['content'] ?? null;
 		}
 
-		return $fallback_content;
+		return strval( $fallback_content );
```

Comment:
🙇

Reply:
I think this is not really adequate, because $fallback_content could be null. But I have a fix in a separate PR